.travis.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     stage: check
     os: linux
     language: python
-    python: 3.7
+    python: 3.8
```

Comment:
It shouldn't replace it, but rather be a new addition.

Reply:
Or was it just a test?

In any case, we've had some issues with conda packages on 3.8, so probably not all dependencies are ready yet.

Reply:
Just a draft, py3.8 has been officially released so should be mergeable soon

Reply:
looks like the main blocker is cython even though it installs in a local py38 env... have you made a start on debugging these issues in another PR @efiop?

Reply:
@casperdcl Not really, 38 is not that urgent :) I would just wait a bit until everything settles in other projects. E.g. https://pypi.org/project/pyarrow/#files still doesn't have 3.8 wheels. I imagine 3.8 will settle down in Jan or something.

Reply:
Def wouldn't spend time on it for now.